mpas_analysis/config.default

```diff
@@ -1417,9 +1421,36 @@ colormapTypeResult = continuous
 # the type of norm used in the colormap
 normTypeResult = linear
 # A dictionary with keywords for the norm
-normArgsResult = {'vmin': -2., 'vmax': 2.}
+normArgsResult = {'vmin': -3., 'vmax': 10.}
 # place the ticks automatically by default
-# colorbarTicksResult = numpy.linspace(-2., 2., 9)
+# colorbarTicksResult = numpy.linspace(-3., 10., 9)
```

Comment:
maybe down to -2 is enough?

Reply:
This was to be consistent with the deeper values, which I want to be symmetric around 0 (separating "cold" from "warm" shelf regions) and which I expanded to -3 to 3.  Also, temperatures under ice shelves get down to -2.5 or below.